src/main/java/org/codelibs/elasticsearch/df/content/csv/CsvContent.java

```diff
@@ -69,15 +69,15 @@ public CsvContent(final Client client, final RestRequest request, final ContentT
         appendHeader = request.paramAsBoolean("append.header", true);
         charsetName = request.param("csv.encoding", "UTF-8");
 
-        final String[] fields = request.paramAsStringArray("fl",
+        final String[] fields = request.paramAsStringArray("header_name",
```

Comment:
We need to keep backward compatibility.
We cannnot remove fl parameter at the moment.